File: test/c14_error.pas

Program example(input, output);

Var x, y : integer;
Books : record
  title: array [100..200] of char;
end;
Function gcd( a, b : integer) : integer;
Begin
  If b = 0 Then gcd := a
  Else gcd := gcd(b, Books)
End;
Begin
  read(x, y);
  write(gcd(x, y))
End.

File: test/c14_error.c
#include<stdio.h>

int x, y; 
struct {
	char title[101]; 
}
Books; 

int gcd(int a, int b){
	if(b == 0){
		return a; 
	}
	else{
		return gcd(b, Books); 
	}
}

int main(int argc, char* argv[]){
	scanf("%d%d", &x, &y); 
	printf("gcd(x, y): %d\n", gcd(x, y)); 
}
